modules/apps/object/object-storage-salesforce/src/main/java/com/liferay/object/storage/salesforce/internal/rest/manager/v1_0/SalesforceObjectEntryManagerImpl.java

```diff
@@ -166,6 +166,11 @@ companyId, getGroupId(objectDefinition, scopeKey),
 			objectDefinition);
 	}
 
+	@Override
+	public String getStorageType() {
+		return ObjectDefinitionConstants.STORAGE_TYPE_SALESFORCE;
```

Comment:
I got your point to avoid the duplication of a reference definition,

but not sure if I will be able to finish it in time, do you mind if I send a follow up pull request?